packages/loader/container-loader/src/container.ts

```diff
@@ -238,20 +230,32 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         });
     }
 
-    public static async create(
+    /**
+     * Create a new container in a detached state.
+     */
+    public static async createDetached(
         loader: Loader,
-        source: DetachedContainerSource,
+        codeDetails: IFluidCodeDetails,
     ): Promise<Container> {
         const container = new Container(
             loader,
             {});
+        await container.createDetached(codeDetails);
+        return container;
+    }
 
-        if (source.create) {
-            await container.createDetached(source.codeDetails);
-        } else {
-            await container.rehydrateDetachedFromSnapshot(source.snapshot);
-        }
-
+    /**
+     * Create a new container in a detached that is initialize with a
+     * snapshot from a previous detached container.
+     */
```

Comment:
nit: detached "state" and initialize -> "initialized"

Reply:
please us to suggest feature to just fix stuff like this :)